lib/errors/NoGetDataSourceMethodDefinedError.ts

```diff
@@ -0,0 +1,8 @@
+export default class NoGetDataSourceMethodDefinedError extends Error {
```

Comment:
as said in slack, we need to check if `name` is used/required, if it is then we can create a `NamedError` as per below. Given the import + extension, going with a single file is enough:

```ts
class NamedError extends Error {
    static readonly message: string = '';

    constructor(message?: string) {
        super(message || new.target.message);
        Object.defineProperty(this, 'name', { value: new.target.name });
    }
}

export class NoGetDataSourceMethodDefinedError extends NamedError {
   static readonly message = 'Method getDataSource not defined';
}

export class NoInitialSdlError extends NamedError {
  static readonly message = 'No initial Sdl';
}
//...
```